docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -401,6 +415,489 @@ docker compose up
+    "status": "pending_user_transfer_start",
+    "message": "waiting on the user to transfer funds",
+    "kyc_verified": "true",
+    "amount_in": {
+      "asset": "stellar:USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5",
+      "amount": "10"
+    },
+    "amount_out": {
+      "asset": "iso4217:USD",
+      "amount": "9.0"
+    },
+    "amount_fee": {
+      "asset": "stellar:USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5",
+      "amount": "1.0"
+    },
+    "memo": "466857256912784999",
+    "memo_type": "id"
+  }
+}
+'
+
+curl localhost:8080/transactions \
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+:::tip
+
+Setting `memo` and `memo_type` is optional. When status is changed to `pending_user_transfer_start`, Anchor Platform sets `memo` and `memo_type` automatically (only if it's not included in the request). Note that the memo must be unique, this is what helps to associate stellar transactions with SEP transaction.
+
+:::
+
+:::note
+
+The Stellar account that will be used to receive funds should be configured in [assets](#specify-your-services-assets)
+
+:::
+
+#### Tracking Stellar Transaction
+
+:::tip
+
+Using Payment Observer allows delegating this step to the Anchor Platform. To enable Payment Observer use `--stellar-observer` flag in the command section of the [compose file](#configuration)
+
+Payment Observer will track all transactions sent to distribution account. When transaction with correct memo is detected in the network, status will automatically change to `pending_anchor` and event will be emitted (if Kafka is used).
+
+:::
+
+:::caution
+
+The Payment Observer won't validate amounts, and it's your responsibility to verify that the amount sent by the user is correct.
+
+:::
+
+First, wait for the transaction to be included in the ledger (using SDK). This transaction must have correct memo and destination address (distribution account). Once this transaction has been detected and verified, notify the user that the funds have been received:
+
+<CodeExample>
+
+```bash
+transaction='
+{
+  "id": '"$transaction_id"',
+  "status": "pending_anchor",
+  "message": "Stellar transaction has been received",
+  "stellar_transactions": [
+    {
+      "id": "<stellar_transaction_id>",
+      "payments": [
+        {
+          "id": "<payment_operation_id>",
+          "amount": {
+            "amount": "10",
+            "asset": "USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5"
+          }
+        }
+      ]
+    }
+  ]
+}
+'
+
+curl localhost:8080/transactions \
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+#### Sending External Transfer
+
+The next step is to send an external transfer, such as a bank transfer, to the user. When done, change status to notify user transfer has been sent, but has not been confirmed yet.
```

Comment:
Lets add that if the business' off-chain rails don't provide enough information on whether the payment is pending instead of delivered, the business should skip this step and mark payments as `completed` after initiating the payment.